src/commands/assets/resolveDID.ts

```diff
@@ -69,6 +105,16 @@ export const resolveDID = async (
     )
   }
 
+  logger.info(chalk.yellowBright(`Metadata:`))
+  logger.info(
+    JSON.stringify(
+      ddo.findServiceByType('metadata').attributes.main,
```

Comment:
maybe this can show all the attributes so that also get the `additionalInformation`

```suggestion
      ddo.findServiceByType('metadata').attributes,
```